5.5/contrib/common.sh

```diff
@@ -86,11 +86,11 @@ function initialize_database() {
   mysql_install_db --datadir=$MYSQL_DATADIR
   start_local_mysql "$@"
 
-  [ -v MYSQL_DISABLE_CREATE_DB ] && return
-
   mysqladmin $admin_flags -f drop test
   mysqladmin $admin_flags create "${MYSQL_DATABASE}"
 
+  [ -v MYSQL_DISABLE_CREATE_DB ] && return
```

Comment:
Yeah, I guess the variable's name no longer makes sense. Will rework this part as I start to figure out how to enable user replication, since the code is that code is there as well.